src/components/new-request/search.vue

```diff
@@ -56,16 +59,16 @@
                         filled
                         label="Select a Jurisdiction"
                         v-model="location">
-                <template slot="item" slot-scope="data">
```

Comment:
This showed an error in VS Code, "Property X does not exist on type Y."

The new format works correctly.